content/sensu-go/5.0/guides/create-read-only-user.md

```diff
@@ -11,49 +11,54 @@ menu:
 ---
 
 ## What is RBAC?
-**Role-Based Access Control** (RBAC) is Sensu's local user management system. RBAC currently supports the management of users and permissions with *organizations*, *environments*, *roles*, and *users*. Support for external directories (such as LDAP) will be available in a future release.
+**Role-based access control** (RBAC) is Sensu's local user management system. RBAC currently supports the management of users and permissions with *namespaces*, *roles*, *users*, and *role bindings*.
 
-## Why use RBAC
+## Why use RBAC?
 RBAC allows you to exercise fine-grained control over how Sensu users interact 
 with Sensu resources. Using RBAC rules, you can easily achieve **multitenancy** 
-so users only have access to resources within their own organization and environment. 
+so different projects and teams can share a Sensu instance. 
 
-## How to create a Read Only user
-The initial installation of Sensu includes a `default` environment, included in the `default` organization, and an `admin` user with full permissions to create, modify, or delete resouces within Sensu. Using `sensuctl`, you can create new roles for users that give as much or as little access as you see fit. The purpose of this guide is to help you create a new user with read-only access to resources within the `default` organization and environment.
+## How to create a read-only user
+By default, Sensu includes a `default` namespace and an `admin` user with full permissions to create, modify, and delete resources within Sensu.
+Using sensuctl configured as the [default `admin` user][2], you can create new roles for users that give as much or as little access as you see fit.
 
-Sensu creates a default read-only role on startup, and you can use that role when creating a read-only user.
+In this section, you'll create a user with read-only access to resources within the `default` namespace.
+Sensu includes a default read-only role called `view` that you can use to create a read-only user.
 
-Create a new user, and apply the read-only role.
+1. Create a user with the username `alice`:
 {{< highlight shell >}}
-$ sensuctl user create 'read-only-user'  --password 'password' --roles 'read-only'
+sensuctl user create alice --password 'password'
 {{< /highlight >}}
 
-You may also apply the read-only role to an existing user account. Users can
-have multiple roles spanning multiple organizations, each with their own set of
-permissions.
+2. Create a `read-only-user` role binding to assign the `view` role to the `alice` user:
+{{< highlight shell >}}
+sensuctl role-binding create read-only-user --role=view --user=alice
```

Comment:
The `view` role is actually a Cluster Role so the flag should be `--cluster-role=view`.